wowchemy/assets/scss/wowchemy/_breadcrumb.scss

```diff
@@ -1,5 +1,5 @@
 .breadcrumb {
-  font-size: 14px;
+  font-size: 0.8rem;
```

Comment:
Depending on the `font_size` chosen by the user, `0.8rem` may be too big (in my case for example I use `XL`, where `1rem = 23px`, and `0.8rem = 18.4px`). The `.article-metadata` [uses `font-size: 14px`](https://github.com/wowchemy/wowchemy-hugo-modules/blob/master/wowchemy/assets/scss/wowchemy/_content.scss#L85) regardless of the `font_size` of the user, that's why I initially used that size for the breadcrumbs. Maybe the `font-size` of `.article-metadata` should be adjusted as well?

Reply:
Thanks for pointing that out Rodri! OK, let's leave the font size the same as `.article-metadata`, 14px, for now.